lib/main/scrapper.js

```diff
@@ -0,0 +1,23 @@
+var cheerio = require('cheerio');
+var http = require('http');
+var url = require('url');
+
+function Scrapper(uri, success_callback, error_callback){
+  if(uri.indexOf('http') < 0){
+    uri = "http://" + uri;
+  }
+  http.get(uri, function(response){
+    var body = '';
```

Comment:
I looked into documentation and can't find a callback which which triggered with complete response. Later I checked `on('data')` and it  really return chunks of page content. 

Already explored `http.get` and `http.request`. Its similar to jQuery `$.get` and `$.ajax`. http.get is direct notation of making `HTTP GET` request.